README.md

```diff
@@ -138,6 +138,15 @@ If you have any questions or discussion topics, start a [GitHub Discussion][8].
 ## Contributing
 Contributions are always welcome. Please read [Contributing.md](CONTRIBUTING.md) if you are considering making contributions.
 
+## Related projects
+- [i3-resurrect](https://github.com/JonnyHaystack/i3-resurrect) - "Simple solution to saving and restoring i3 workspaces"
+- [tmux-resurrect](https://github.com/tmux-plugins/tmux-resurrect)
+- [tmux-continuum](https://github.com/tmux-plugins/tmux-continuum) - an excellent companion to
+tmux-resurrect
+- [qutebrowser](https://github.com/qutebrowser/qutebrowser) - which has excellent session
+management, especially if you create bindings for saving and loading individual windows
+- [firefox-i3-workspaces](https://github.com/yurikhan/firefox-i3-workspaces) - to restore Firefox windows to the same workspaces, helper browser extension is needed to install
```

Comment:
Can this also be moved to the Similar Software section in the develop branch? Here's the description that can be used (make sure to update the table of contents as well).

```markdown
### firefox-i3-workspaces
[Firefox-i3-workspaces](https://github.com/yurikhan/firefox-i3-workspaces) is a Firefox plugin and Python script to restore multiple windows
of Firefox on i3.
```